src/base/matrix_vec.rs

```diff
@@ -9,7 +9,7 @@ use base::allocator::Allocator;
 use base::default_allocator::DefaultAllocator;
 use base::dimension::{Dim, DimName, Dynamic, U1};
 use base::storage::{ContiguousStorage, ContiguousStorageMut, Owned, Storage, StorageMut};
-use base::Scalar;
+use base::{Scalar, Vector};
```

Comment:
This will be needed by the suggestion bellow.

```suggestion
use base::{Scalar, Vector};
use base::constraint::{SameNumberOfRows, ShapeConstraint};
```